lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -56,6 +56,7 @@ class GradingPolicySerializer(serializers.Serializer):
     def get_assignment_policies(self, grading_policy):
         return [{
             'num_droppable': assignment_policy['drop_count'],
+            'num_total': assignment_policy['min_count'],
```

Comment:
It's called min_count? What is this world?

Reply:
This is a required field in Studio, and the value must be greater than 0. Do you think we should be more defensive here?

Reply:
Nah, as long as our system is enforcing it before this point, we are safe to treat it as a valid value.